src/Modules/Windows-Full/Microsoft.PowerShell.ODataUtils/Microsoft.PowerShell.ODataUtils.psd1

```diff
@@ -22,7 +22,7 @@ Author = 'Microsoft Corporation'
 CompanyName = 'Microsoft Corporation'
 
 # Copyright statement for this module
-Copyright = '(c) 2014 Microsoft. All rights reserved.'
+Copyright (c) Microsoft. All rights reserved.'
```

Comment:
Typo. Please revert "Copyright =".

I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."